### DISCUSSION TITLE
Alerting and Actions

### DISCUSSION THREAD

**Discussion by kdeiss** [telekom-security/tpotce#793] [General]
First of all, I would like to thank you very much for this powerful tool.

I think the discussion here (https://github.com/telekom-security/tpotce/discussions/745) is going into a direction that moves me too

If that is not possible via Kibana, do you see another possibility to intercept certain events and carry out warning actions?

regards 

Klaus Deiss

**Comment by Clankcoll** [comment]
Hi i know i am pretty late but maybe this will help you [204](https://github.com/telekom-security/tpotce/issues/204).

I am not sure what the best Practice would be but i am looking forward to start it this week and may then come back to infrom you with my results.

Kind regards
C
